### PR TITLE
(core{01-03}.dev) Network Config / Variant / Tests

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Cut'n'pasting the same config for multiple hosts is a sign that common config needs to be factored out. The existing configuration being used for EL9 dev hypervisors is at `site/dev/role/hypervisor/osfamily/RedHat/major/9.yaml`.  These changes should probably either be included there (which would be problematic as this role is being used for multiple hardware configurations) or replaced with a `site/dev/variant/*` .

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The variant looks reasonable but tests are needed (note that the PR and commit title says config and tests :smile: ).

As these are critical hosts, we should add examples to all 3 of the nodes.  Here is an existing `hosts/nodes` that tests variant networking that should be a reasonable template:

https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/lsstcam-dc01.ls.lsst.org_spec.rb

File: hieradata/site/dev/variant/1114S-rev_1.01.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+profile::nm::connections:
+  enp1s0f1:
```
Comment (jhoblitt) [inline hieradata/site/dev/variant/1114S-rev_1.01.yaml:3]: Please add spec on each host for these interfaces.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Comment by jhoblitt** [comment]
Note that https://github.com/lsst-it/lsst-control/pull/868 has been merged and the hiera key `profile::nm::connections` is no longer being  used / will need to be updated.

File: hieradata/site/dev/variant/1114S-rev_1.01.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/site/dev/variant/1114S-rev_1.01.yaml:3]: Hi @jhoblitt, I added the spec for each of the interfaces but I'm having issues with the nm connections. I would appreciate your help if you could help revise the code based on the logs below.

Comment (dtapiacl) [reply hieradata/site/dev/variant/1114S-rev_1.01.yaml:3]: Updated and fixed tests. PR should be mergeable now.

**Review by jhoblitt — APPROVED** [review]
LGTM. Thanks for persisting.

**Pull request merged**